src/PKSim.Infrastructure/ORM/Repositories/ParametersInContainerRepository.cs

```diff
@@ -43,19 +30,23 @@ public ParameterMetaData ParameterMetaDataFor(IParameter parameter)
          if (parameter == null)
             return null;
 
-         return ParameterMetaDataFor(_entityPathResolver.PathFor(parameter));
-      }
+         if (parameter.IsDistributed())
+            return parameterMetaDataFrom(_parameterDistributionRepository, parameter);
 
-      protected override void DoStart()
-      {
-         base.DoStart();
-         _parameterMetaDataList.Each(x => _parameterMetaDataCache.Add($"{x.ParentContainerPath}{ObjectPath.PATH_DELIMITER}{x.ParameterName}", x));
+         if (!parameter.Formula.IsConstant())
+            return parameterMetaDataFrom(_parameterRateRepository, parameter);
```

Comment:
will it also work for parameters, which are defined as rates but have a constant formula (e.g. "0" or "1")?

Reply:
See above. This strategy does not work. If we want to be able to retrieve ParmaeterMEtaData from a PArameter, we will have to think about something else. This function is new and at the moment only used by the ResetParameterCommand.